lib/tests/test_rcptid.c

```diff
@@ -36,6 +37,7 @@ PersistState *state;
 void
 setup_persist_id_test(void)
 {
+  app_startup();
```

Comment:
The `teardown_persist_id_test` should call the `app_shutdown`.

Reply:
add missing app_shutdown: done